### PR TITLE
feat: zset cmd zrevrangebyscore

### PR DESCRIPTION
**Pull request by callme-taota**
image

pass test

https://github.com/OpenAtomFoundation/pikiwidb/issues/30

### CODE REVIEW

**Comment by 578223592** [comment]
@callme-taota  zadd is merged.
https://github.com/OpenAtomFoundation/pikiwidb/pull/143

**Comment by callme-taota** [comment]
> @callme-taota zadd is merged. #143

update

**Review by DeepBunny — DISMISSED** [review]

File: src/cmd_zset.cc [hunk 1]
```diff
@@ -217,4 +217,86 @@ void ZRangebyscoreCmd::DoCmd(PClient* client) {
...
+ZRevRangeByScoreCmd::ZRevRangeByScoreCmd(const std::string& name, int16_t arity)
+    : BaseCmd(name, arity, kCmdFlagsWrite, kAclCategoryWrite | kAclCategorySortedSet) {}
+
+bool ZRevRangeByScoreCmd::DoInitial(PClient* client) {
+  client->SetKey(client->argv_[1]);
+  return true;
+}
+
+void ZRevRangeByScoreCmd::DoCmd(PClient* client) {
+  double min_score = 0, max_score = 0;
```
Comment (AlexStocks) [inline src/cmd_zset.cc]: pikiwidb 没有一行定义多个变量的风格，一行只定义一个变量。其他地方比照处理下哈

File: src/cmd_zset.cc [hunk 2]
```diff
@@ -217,4 +217,86 @@ void ZRangebyscoreCmd::DoCmd(PClient* client) {
...
+      client->AppendContent(buf);
+    }
+  } else {
+    client->AppendArrayLen(count);
+    for (; start < end; start++) {
+      client->AppendStringLenUint64(score_members[start].member.size());
+      client->AppendContent(score_members[start].member);
+    }
+  }
+  return;
```
Comment (LuckyTom1024) [inline src/cmd_zset.cc]: 这个return应没必要吧，直接去掉

**Review by 578223592 — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**